bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_customizecards.dat

```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card mass 25 125
+set param_card yukawa 6 -172.5
```

Comment:
could you please elaborate what's the meaning of negative yukawa coupling

Reply:
removed -ve sign.

Reply:
it seems there is still negative yukuwa coupling